src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/adminappealwithdrawn/AdminAppealWithdrawnAboutToSubmitHandler.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.reform.sscs.ccd.presubmit.adminappealwithdrawn;
+
+import static java.util.Objects.requireNonNull;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.sscs.ccd.callback.Callback;
+import uk.gov.hmcts.reform.sscs.ccd.callback.CallbackType;
+import uk.gov.hmcts.reform.sscs.ccd.callback.PreSubmitCallbackResponse;
+import uk.gov.hmcts.reform.sscs.ccd.domain.EventType;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+import uk.gov.hmcts.reform.sscs.ccd.presubmit.PreSubmitCallbackHandler;
+
+
+@Service
+@Slf4j
+public class AdminAppealWithdrawnAboutToSubmitHandler implements PreSubmitCallbackHandler<SscsCaseData> {
```

Comment:
For the future, when you have to set a field you don't need to create new handlers. You can reuse this handler: EventToFieldPreSubmitCallbackHandler.